llvm/include/llvm/SYCLLowerIR/SYCLVirtualFunctionsAnalysis.h

```diff
@@ -0,0 +1,28 @@
+//===--- SYCLVirtualFunctionsAnalysis.cpp ---------------------------------===//
```

Comment:
nitpick you can definitely ignore: the number of `-` on the left and right side is greatly unbalanced and looks weird to me

Reply:
The balance should be restored by 57e43cf0fd010cd160fc9ebe9073b7c7c2711cc9

Reply:
all is right with the world